### ISSUE TITLE
Missing the license header for GPL-3.0

### ISSUE THREAD

**Issue by chenrui333** [hadolint/hadolint#460]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

👋  Per [SPDX license index](https://spdx.org/licenses), `gpl-3.0` is the deprecated license, adding a license header would further clarify the license application and solve my issue. Thanks.

Kind of relates to https://github.com/Homebrew/homebrew-core/pull/59739

Cheers.

**Comment by SuperSandro2000** [comment]
I don't think this is a bug. This is merely a convince. The LICENSE file clearly states it is 3.0 and not 3.0 or later. Also 3.0 is not deprecated but SPDX wants you to clarify if it is 3.0-only or 3.0-or-later.

Homebrew could also just parse the LICENSE file or ask the GitHub Api.

**Comment by m-ildefons** [comment]
The license is clearly specified in the LICENSE file and it looks like homebrew now correctly recognizes it as well:

<img width="1522" height="900" alt="Image" src="https://github.com/user-attachments/assets/688645cd-890a-4b3f-9219-9cd5c3405ce6" />

Closing this issue.
